telegram/utils/helpers.py

```diff
@@ -139,3 +144,25 @@ def effective_message_type(entity):
             return i
 
     return None
+
+
+def passport_auth_url(bot_id, scope, public_key, payload, callback_url=None):
+    """
+    Creates a passport auth url that asks the users for the specified fields
+
+    Args:
+        bot_id (:obj:`int`): Unique identifier for the bot. Can be extracted from the bot token.
+        scope (List[:obj:`str`]): List the names of fields you want to access.
+        public_key (:obj:`str`): Public key of your bot.
+        payload (:obj:`str`): Bot-specified payload
+        callback_url (:obj:`str`, optional): URL to which the user will be redirected.
+
+    """
+    url = 'tg://resolve?domain=telegrampassport' + \
```

Comment:
A tg://resolve url is no longer clickable on Telegram. We can use t.me/telegrampassport to send a clickable url with the same functionality on telegram.

Reply:
Unfortunately, no. The t.me/telegrampassport url seems more like a bug in the username resolve process of Telegram Desktop. I already contacted Telegram Support and the bot Support. Telegram support didn't know anything about why the links are no longer clickable. Bot support did not reply yet.

Reply:
@code1mountain Any updates on this?

Reply:
I'm just gonna remove the function for now. We can always add it in the future if we figure out how to do it properly.
(this should be okay since we haven't released it yet)